src/Data/List/NonEmpty/Relation/Unary/Any.agda

```diff
@@ -0,0 +1,49 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Non-empty lists where at least one element satisfies a given property
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.List.NonEmpty.Relation.Unary.Any where
+
+open import Data.List.NonEmpty using (List⁺; _∷_)
+
+open import Data.List.Relation.Unary.Any as List using (here; there)
+open import Data.List.Base using ([]; _∷_)
+open import Data.List.NonEmpty.Base using (List⁺; _∷_; toList)
+open import Data.Product.Base using (_,_)
+open import Level using (Level; _⊔_)
+open import Relation.Unary using (Pred; Satisfiable; _⊆_)
+
+private
+  variable
+    a p : Level
+    A : Set a
+    P Q : Pred A p
```

Comment:
I seem to recall you objecting to this kind of `variable` declaration before, but happy if they are now allowable ;-)

Reply:
I was persuaded otherwise several years back I think!